ttnn/cpp/ttnn/operations/eltwise/binary_backward/binary_backward_pybind.hpp

```diff
@@ -402,10 +403,168 @@ Keyword args:
             py::arg("memory_config") = std::nullopt});
 }
 
+template <typename binary_backward_operation_t>
+void bind_binary_bw_operation(py::module& module, const binary_backward_operation_t& operation, const std::string& description) {
+    auto doc = fmt::format(
+        R"doc({0}(input_tensor_a: ttnn.Tensor, input_tensor_b: Union[ttnn.Tensor, int, float], *, memory_config: Optional[ttnn.MemoryConfig] = None, dtype: Optional[ttnn.DataType] = None, activations: Optional[List[str]] = None) -> ttnn.Tensor
```

Comment:
Also a minor Update here : `input_tensor_a` and  `input_tensor_b` can be of `ComplexTensor `.